skafka/src/main/scala/com/evolutiongaming/skafka/SaslSupportConfig.scala

```diff
@@ -0,0 +1,99 @@
+package com.evolutiongaming.skafka
+
+import cats.implicits._
+import com.evolutiongaming.config.ConfigHelper._
+import com.evolutiongaming.skafka.ConfigHelpers._
+import com.typesafe.config.{Config, ConfigValue}
+import org.apache.kafka.common.config.SaslConfigs
+
+import java.nio.file.Path
+import scala.concurrent.duration.{DurationLong, FiniteDuration}
+
+final case class SaslSupportConfig(
+  kerberosServiceName: Option[String]          = None,
+  kerberosKinitCmd: Path                       = Path.of(SaslConfigs.DEFAULT_KERBEROS_KINIT_CMD),
```

Comment:
we need to rework, I'd like to avoid potentially heavy side-effecting code executed in scope fo default case class construction, any suggestions?

Reply:
I see three options.

1. Late initialization of default object's members. When config will be created, then only default values for uninitialized members will be created, not all of them. 
2. Do not use defaults at all. All members of `SaslSupportConfig` can be Optional. 
3. Also I can make `SaslSupportConfig` and its default object simpler. There are two non-trivial types used - Path and FiniteDuration. I can use String and Long respectively. But it lowers readability and removes validation for these members.

Reply:
`FiniteDuration` - is ok,

`Path.of` - is not ok, I'd suggest using plain `String` there, if you want - custom `case class` as type wrapper over `String`

Reply:
I've moved from Path to String.